daemon/kill.go

```diff
@@ -179,7 +179,9 @@ func (daemon *Daemon) Kill(container *containerpkg.Container) error {
 // killPossibleDeadProcess is a wrapper around killSig() suppressing "no such process" error.
 func (daemon *Daemon) killPossiblyDeadProcess(container *containerpkg.Container, sig int) error {
 	err := daemon.killWithSignal(container, sig)
+	logrus.WithError(err).WithField("pid", container.State.Pid).Error("KILLWITHSIGNAL")
 	if errdefs.IsNotFound(err) {
+		logrus.WithError(err).WithField("pid", container.State.Pid).Error("ISNOTFOUND")
```

Comment:
Looks like these were temporary for debugging? Are they still needed? (I see there's a debug line below already if there's an error; if it's still useful, probably better to combine